GOVERNANCE.md

```diff
@@ -214,44 +214,42 @@ Pull requests shall be tagged in GitHub as described in discussion #2789 (https:
 * Minor production releases to introduce enhancements should be combined  to minimize the number of production releases, targeting minor production releases to be issued around four weeks or so as long as there is a queue of approved PRs.  (This frequency can be increased in times of urgent need for new functionality).
   * *Rationale:*   Minimizing the number of production releases will help with supportability, by reducing the number of releases that end users wishing to remain current need to consider, and reducing communications overhead.
-* Each Minor Production Release shall require the approval of two maintainers.
-* Minor Production releases shall be reported in brief to the CRWG and the SWG, 
-* A roadmap of anticipated minor releases shall be reported by the maintainers to the CRWG based on PRs that are in process.
+* Each minor production release shall require the approval of two maintainers.
+* Minor production releases shall be reported in brief to the CRWG and the SWG, 
+* A roadmap of anticipated minor production release shall be reported by the maintainers to the CRWG based on PRs that are in process.
 
 ## 9.4  Production Patch Release Scheduling and Approvals
 
-* Patch releases to correct defects without releasing new functionality may be scheduled by the maintainers based on the presence of approves defect correction PRs, or other non-functional PRs (e.g. security remediations).
-*  Patch releases require the approval of one maintainer
-*  Patch releases shall be reported to the CRWG.
+* Production patch releases to correct defects without releasing new functionality may be scheduled by the maintainers based on the presence of approved defect correction PRs, or other non-functional PRs (e.g. security remediations).
+* Production patch releases require the approval of one maintainer
+* Production patch releasess shall be reported to the CRWG.
 
  ## 9.5 Summary of Release Approval Requirements
 
- | Type of Release        | Approval Requirement |  Notes                    |
-| ----------------------- | -------------------- |  -------------------------------------------  |
-| Development Release     | 1 maintainer         | Scheduling is up to the maintainer            |
-| Major Release (6.0.0)   | 2 maintainers         | Scheduling via SWG; Include analysis of the changes from last major release as part of the approval  |
-| Minor Release (6.1.0)   | 2 maintainers        | Scheduling is up to the maintainers, but aim to keep to around every 4 weeks and no more than fortnightly   |
-| Patch Release (6.1.1)    | 1 maintainer         | Scheduling is up to the maintainer            |
-
-
+| Type of Release      | Approval Requirement |  Notes                                        |
+| ---------------------| -------------------- |  -------------------------------------------  |
+| Development Release  | 1 maintainer         | Scheduling is up to the maintainer            |
```

Comment:
thought we'd be adding an example of a Development Release too e.g. (6.0.0-dev.13)

Reply:
yes, Eteri can you please do that>

Reply:
Can we remove the first row for "Development Release" please, as we now have it at the bottom of the table instead with the example syntax too 👍